README.rst

```diff
@@ -15,11 +15,15 @@ of the latest code: http://github.com/joblib/joblib/archives/master
 Installing
 =========================
 
-As any Python packages, to install joblib, simply do::
+As most of pure Python packages, to install joblib, simply do::
 
-    python setup.py install
+    pip install joblib
 
-in the source code directory.
+from any directory or
+
+    pip install .
```

Comment:
I would keep python setup.py install. Not sure what `pip install .` brings over `python setup.py install`. I think pip actually looks at setup.py and uses the info there.